tests/release/release.go

```diff
@@ -0,0 +1,133 @@
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	// klog "k8s.io/klog/v2"
+	"knative.dev/pkg/apis"
+)
+
+var snapshotComponents = []gitopsv1alpha1.ApplicationSnapshotComponent{
+	{"component-1", "quay.io/redhat-appstudio/component1@sha256:d5e85e49c89df42b221d972f5b96c6507a8124717a6e42e83fd3caae1031d514"},
+	{"component-2", "quay.io/redhat-appstudio/component2@sha256:a01dfd18cf8ca8b68770b09a9b6af0fd7c6d1f8644c7ab97f0e06c34dfc5860e"},
+	{"component-3", "quay.io/redhat-appstudio/component3@sha256:d90a0a33e4c5a1daf5877f8dd989a570bfae4f94211a8143599245e503775b1f"},
+}
+
+var _ = framework.ReleaseSuiteDescribe("test-release-service-happy-path", Label("release"), func() {
+	defer GinkgoRecover()
+	// Initialize the tests controllers
+	framework, err := framework.NewFramework()
+	Expect(err).NotTo(HaveOccurred())
+
+	var devNamespace string
+	var managedNamespace string
+
+	var _ = Describe("test-release-service-happy-path", func() {
+		BeforeAll(func() {
+			// Recreate random namespaces names per each test because if using same namespace names, the next test will not be able to create the namespaces as they are terminating
+			devNamespace = "user-" + uuid.New().String()
+			managedNamespace = "managed-" + uuid.New().String()
+
+			// Create the dev namespace
+			_, err := framework.CommonController.CreateTestNamespace(devNamespace)
+			Expect(err).NotTo(HaveOccurred(), "Error when creating namespace '%s': %v", devNamespace, err)
+
+			// Create the managed namespace
+			_, err = framework.CommonController.CreateTestNamespace(managedNamespace)
+			Expect(err).NotTo(HaveOccurred(), "Error when creating namespace '%s': %v", managedNamespace, err)
+		})
+
+		AfterAll(func() {
+			// Delete the dev and managed namespaces with all the resources created in them
+			Expect(framework.CommonController.DeleteNamespace(devNamespace)).NotTo(HaveOccurred())
+			Expect(framework.CommonController.DeleteNamespace(managedNamespace)).NotTo(HaveOccurred())
+		})
+
+		var _ = Describe("Creation of the 'Happy path' resources", func() {
+			It("Create an ApplicationSnapshot.", func() {
+				_, err := framework.ReleaseController.CreateApplicationSnapshot(snapshotName, devNamespace, applicationName, snapshotComponents)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			It("Create Release Strategy", func() {
+				_, err := framework.ReleaseController.CreateReleaseStrategy(releaseStrategyName, managedNamespace, releasePipelineName, releasePipelineBundle, releaseStrategyPolicy, releaseStrategyServiceAccount)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			It("Create ReleasePlan in dev namespace", func() {
+				AutoReleaseLabel := ""
+				_, err := framework.ReleaseController.CreateReleasePlan(sourceReleasePlanName, devNamespace, applicationName, managedNamespace, AutoReleaseLabel)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			It("Create ReleasePlanAdmission in managed namespace", func() {
+				AutoReleaseLabel := ""
```

Comment:
why not just pass `""` into the function on the next line and not declare this variable?

Reply:
Same comment as above